packages/plugin-data/test/api/data/soql/queryFields.test.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import { expect } from 'chai';
+
+import sinon = require('sinon');
+import { retrieveColumns } from '../../../../src/api/data/soql/queryFields';
+import * as TestUtil from '../../../testUtil';
+import { queryFieldsExemplars } from './test-files/queryFields.exemplars';
+
+chai.use(chaiAsPromised);
+
+describe('queryFields tests', () => {
+  // let toolingSpy: sinon.SinonSpy;
+  // let querySpy: sinon.SinonSpy;
+  const fakeConnection = TestUtil.createFakeConnection();
+  let sandbox: any;
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
```

Comment:
sandboxes only need to be created once.  The sandbox state is restored in afterEach.